ecf/aqm/stats/jevs_aqm_stats.ecf

```diff
@@ -39,7 +39,7 @@ module load prod_envir/${prodenvir_ver}
 
 module list
 
-export cyc
+export cyc=23
```

Comment:
I don't think we want the "=23"

Reply:
You are absolutely correct!  I just had that in there for testing.  I'll remove it.